py/embedded-server/deephaven_server/server.py

```diff
@@ -120,22 +119,35 @@ def __init__(
         port: Optional[int] = None,
         jvm_args: Optional[List[str]] = None,
         extra_classpath: Optional[List[str]] = None,
+        default_jvm_args: Optional[List[str]] = None,
     ):
         """
         Creates a Deephaven embedded server. Only one instance can be created at this time.
+
+        Args:
+            host (Optional[str]): The host to bind the server to, defaults to None, meaning local host.
+            port (Optional[int]): The port to bind the server to, defaults to None. When None, if a user defined
+                configuration file is present and a port number is specified in it, use that port, otherwise, use the
+                default 10000. Refer to the Deephaven documentation for more information on the configuration file.
+            jvm_args (Optional[List[str]]): The common, user specific JVM arguments, such as JVM heap size, the
+                authentication handler to use, and other related JVM options. Defaults to None.
```

Comment:
I don't think we should recommend setting authentication handler via jvm args; it's really meant to be a Deephaven configuration property, in which case using the configuration file is the preferred way IMO